src/openrct2/ride/Ride.cpp

```diff
@@ -7225,7 +7225,7 @@ TileElement* get_station_platform(int32_t x, int32_t y, int32_t z, int32_t z_tol
             if (!tileElement->AsTrack()->IsStation())
                 continue;
 
-            if (z - z_tolerance > tileElement->base_height || z + z_tolerance < tileElement->base_height)
+            if (z - z_tolerance > tileElement->GetBaseZ() || z + z_tolerance < tileElement->GetBaseZ())
```

Comment:
This again isn't an issue with this pr but might be on the other. Z_tolerance is not the same as clearanceZ which is what rangedZ is expecting.

Reply:
I don't get what you think needs or should be changed here. I get that `z_tolerance != clearanceZ`, so maybe we just shouldn't use `CoordsXYRangedZ` there?

Reply:
I see you commented there now, nvm

Reply:
Just to make sure: are `z` and `z_tolerance` already big Zs here?